app/api/helpers/mail.py

```diff
@@ -725,3 +725,45 @@ def send_email_to_moderator(video_stream_moderator):
             settings=get_settings(),
         ),
     )
+
+
+def send_email_after_event(email, event):
```

Comment:
you have to add the functions in schedule jobs too. please test PR completely. If you have not setUp email thing. Just use logger and see if it's working.

Reply:
I tried testing using logger, but there is no log regarding the after event mail. How do I set up the SendGrid ?
I got the API key from SendGrid. Do I need to create a sender ID too??

For the logger, I just have to set up celery and redis??